cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -255,7 +263,7 @@ func (c Client) WaitForClusterAvailable(retries int, retryInterval time.Duration
 			fmt.Printf("[util/etcd] Waiting %v until next retry\n", retryInterval)
 			time.Sleep(retryInterval)
 		}
-		fmt.Printf("[util/etcd] Attempting to see if all cluster endpoints are available %d/%d\n", i+1, retries)
+		fmt.Printf("[util/etcd] Attempting to see if all cluster endpoints (%s) are available %d/%d\n", c.Endpoints, i+1, retries)
```

Comment:
This change has the potential to spew a lot of endpoints in a regular message here. I would recommend moving the endpoint dump to a separate `klog.V(2).Infof(...)` message right after this one.